sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py

```diff
@@ -587,8 +512,8 @@ def append_data(self, data,  # type: Union[bytes, str, Iterable[AnyStr], IO[AnyS
         :keyword ~azure.storage.filedatalake.CustomerProvidedEncryptionKey cpk:
             Encrypts the data on the service-side with the given key.
             Use of customer-provided keys must be done over HTTPS.
-        :returns: dict of the response header.
-        :rtype: dict[str, str], dict[str, ~datetime.datetime], or dict[str, int]
```

Comment:
These are inaccurate? Just making sure

Reply:
These aren't inaccurate, it's just that all of the existing response dicts are all of type `Dict[str, Any]` as the keys can really be any of the three values above so making that consistent here.

Let me know how you feel about this.